### PR TITLE
Covered reminder for EmberConf 2020 CfP (Times 125)

### PR DESCRIPTION
**Pull request by ijlee2**

### CODE REVIEW

**Comment by amyrlam** [comment]
ignore red travis - let's rebase the parent blog branch https://github.com/ember-learn/ember-blog/pull/353 after this is merged in

File: source/2019-11-22-the-ember-times-issue-125.md [hunk 1]
```diff
@@ -24,13 +24,11 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Reminder: EmberConf 2020 CFP, Due December 1 🎤](https://cfp.emberconf.com/events/emberconf-2020/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Each year, [EmberConf](https://emberconf.com/) gets to feature wonderful, *ambitious* talks and workshops. We encourage you to take stage in 2020 and [share what you learned (or want to learn) with the world](https://cfp.emberconf.com/events/emberconf-2020/)! You can give a 5, 15, or 30-min talk, or a 3-hour workshop. You can also submit more than 1 proposal.
```
Comment (jaredgalanis) [inline source/2019-11-22-the-ember-times-issue-125.md]:
```suggestion
Each year, [EmberConf](https://emberconf.com/) gets to feature wonderful, *ambitious* talks and workshops. We encourage you to take to the stage in 2020 and [share what you learned (or want to learn) with the world](https://cfp.emberconf.com/events/emberconf-2020/)! You can give a 5, 15, or 30-min talk, or a 3-hour workshop. You can also submit more than 1 proposal.
```

**Review by jaredgalanis — APPROVED** [review]
Reads great! Just one small suggestion.

File: source/2019-11-22-the-ember-times-issue-125.md (hunk 1 above)
Comment (ijlee2) [reply source/2019-11-22-the-ember-times-issue-125.md]: Thanks! It seems the saying is "take the stage," so let me change to that.

File: source/2019-11-22-the-ember-times-issue-125.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Reminder: EmberConf 2020 CFP, Due December 1 🎤](https://cfp.emberconf.com/events/emberconf-2020/)
+
+Each year, [EmberConf](https://emberconf.com/) gets to feature wonderful, *ambitious* talks and workshops. We encourage you to take stage in 2020 and [share what you learned (or want to learn) with the world](https://cfp.emberconf.com/events/emberconf-2020/)! You can give a 5, 15, or 30-min talk, or a 3-hour workshop. You can also submit more than 1 proposal.
```
Comment (ijlee2) [inline source/2019-11-22-the-ember-times-issue-125.md]:
```suggestion
Each year, [EmberConf](https://emberconf.com/) gets to feature wonderful, *ambitious* talks and workshops. We encourage you to take the stage in 2020 and [share what you learned (or want to learn) with the world](https://cfp.emberconf.com/events/emberconf-2020/)! You can give a 5, 15, or 30-min talk, or a 3-hour workshop. You can also submit more than 1 proposal.
```

**Pull request merged**